docker/init.sls

```diff
@@ -11,34 +11,39 @@ include:
 docker-package-dependencies:
   pkg.installed:
     - pkgs:
-      {%- if grains['os_family']|lower == 'debian' %}
-      - apt-transport-https
-      - python-apt
-      {%- endif %}
-      - iptables
-      - ca-certificates
-      {% if docker.kernel.pkgs is defined %}
-        {% for pkg in docker.kernel.pkgs %}
-        - {{ pkg }}
-        {% endfor %}
-      {% endif %}
+       {%- for pkgname in docker_packages %}
+      - {{ pkgname }}
+       {%- endfor %}
+       {%- if "pip" in docker and "pkgname" in docker.pip %}
+      - {{ docker.pip.pkgname }}
+       {%- endif %}
     - unless: test "`uname`" = "Darwin"
+    - refresh: {{ docker.refresh_repo }}
 
 docker-package:
   pkg.installed:
     - name: {{ docker_pkg_name }}
-    - version: {{ docker_pkg_version }}
-      {%- if grains['os']|lower not in ('amazon', 'fedora', 'suse',) %}
-      - pkgrepo: docker-package-repository
-      {%- endif %}
+    - version: {{ docker_pkg_version or 'latest' }}
     - refresh: {{ docker.refresh_repo }}
     - require:
       - pkg: docker-package-dependencies
       {%- if grains['os']|lower not in ('amazon', 'fedora', 'suse',) %}
-      - pkgrepo: docker-package-repository
+    - pkgrepo: docker-package-repository
       {%- endif %}
     - require_in:
       - file: docker-config
+       {%- if grains.os_family in ('Suse',) %}   ##workaround https://github.com/saltstack-formulas/docker-formula/issues/198
+  cmd.run:
+    - name: /usr/bin/pip install {{ '--upgrade' if docker.pip.upgrade else '' }} pip
+       {%- else %}
+  pip.installed:
+    - name: pip
+    - reload_modules: true
+    - upgrade: {{ docker.pip.upgrade }}
+       {%- endif %}
```

Comment:
The Jinja block indenting doesn't match the existing indent (above).

Reply:
Thanks.